### PR TITLE
Auto Changes (New Drivebase paths)

### PR DESCRIPTION
**Pull request by stewluj**

### CODE REVIEW

File: src/main/java/frc/team2412/robot/Subsystems.java [hunk 1]
```diff
@@ -18,11 +18,11 @@
 public class Subsystems {
 	public static class SubsystemConstants {
 		public static final boolean DRIVEBASE_ENABLED = true;
-		public static final boolean ARM_ENABLED = true;
-		public static final boolean INTAKE_ENABLED = true;
+		public static final boolean ARM_ENABLED = false;
+		public static final boolean INTAKE_ENABLED = false;
```
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/Subsystems.java:22]: I'd recommend setting these back to `true` so we don't run into the same issue we had today.

File: src/main/java/frc/team2412/robot/util/auto/AutonomousChooser.java [hunk 2]
```diff
@@ -14,21 +14,32 @@ public interface CommandSupplier {
 
 	public AutonomousChooser() {
 		autonomousModeChooser.setDefaultOption(
-				"LeftLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("LeftLeaveCom"));
+				"FlatLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("LeftLeaveCom"));
 		autonomousModeChooser.addOption(
-				"RightLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("RightLeaveCom"));
+				"BumpLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("RightLeaveCom"));
```
Comment (TAKBS2412) [inline src/main/java/frc/team2412/robot/util/auto/AutonomousChooser.java:19]: Just a note that the names aren't always matching here (as we found out last meeting).

**Comment by KangarooKoala** [comment]
Are there any changes that weren't made in #66?

**Comment by TAKBS2412** [comment]
> Are there any changes that weren't made in #66?

I think just `rename auto paths to reflect file names`, although the PR is so old we might be better off closing this one and doing that elsewhere.

**Comment by KangarooKoala** [comment]
Yeah, also `rename auto paths to reflect file paths` (`6a56dd3`) seems to have been included in `fullAuto` anyways.

**Comment by KangarooKoala** [comment]
All changes seem to have been included in the fullAuto PR

**Pull request closed without merging**